plug.vim

```diff
@@ -2028,11 +2047,12 @@ function! s:git_validate(spec, check_branch)
               \ branch, a:spec.branch)
       endif
       if empty(err)
-        let [ahead, behind] = split(s:lastline(s:system(printf(
+        let [output, shellerror] = s:system(printf(
               \ 'git rev-list --count --left-right HEAD...origin/%s',
-              \ a:spec.branch), a:spec.dir)), '\t')
-        if !v:shell_error && ahead
-          if behind
+              \ a:spec.branch), a:spec.dir)
+        let [ahead; behind] = split(s:lastline(output), '\t')
+        if !shellerror && ahead
+          if len(behind) > 0
```

Comment:
The original code expected exact two numbers, which is a correct assumption given that `git rev-list --count --left-right HEAD...origin/master` works as expected. I think `behind[0]` should suffice. What do you think?